tests/versioned/attributes-tests.js

```diff
@@ -379,7 +379,9 @@ function createAttributesTests(t) {
         'graphql.field.args.blee': 'second'
       }
       const resolveAttributes = resolveHelloSegment.attributes.get(SEGMENT_DESTINATION)
+      const operationAttributes = operationSegment.attributes.get(SEGMENT_DESTINATION)
       t.matches(resolveAttributes, expectedArgAttributes)
+      t.matches(operationAttributes, expectedArgAttributes)
```

Comment:
I called this out when we reviewed but this file is only executed for apollo-sever and apollo-sever-express. You'll want to add the same assertion in all the attirbutes tests for every framework and federation